packages/tooling/fast-tooling-react/src/form/custom-controls/css/color-picker-component/color-picker.ts

```diff
@@ -0,0 +1,400 @@
+import { ColorHSV, ColorRGBA64, hsvToRGB, parseColor, rgbToHSV } from "@microsoft/fast-colors";
+import { attr, DOM, observable } from "@microsoft/fast-element";
+import { isNullOrWhiteSpace } from "@microsoft/fast-web-utilities";
+import { isNumber } from "util";
```

Comment:
Doesn't look like this gets used (also util is not a valid lib in browser).

```suggestion
```

Reply:
I think this was auto-added at some point. Tools trying to be "helpful". Removed.